docfx_yaml/extension.py

```diff
@@ -24,6 +24,11 @@
 import copy
 import shutil
 import black
+
+import logging
```

Comment:
Put this with the other imports. Move the `getlogger` call to below the imports.

Reply:
Done.